haiku-misc/universalscroller/patches/universalscroller-4.0~git.patchset

```diff
@@ -0,0 +1,59 @@
+From 840404a338fde55c779a620fefec961c1ddd4396 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
+Date: Mon, 23 Apr 2018 21:16:24 +0200
+Subject: Build fixes
+
+
+diff --git a/Makefile b/Makefile
+index c72660f..a36af23 100644
+--- a/Makefile
++++ b/Makefile
+@@ -59,8 +59,8 @@ dist: all
+ 	scripts/substituting_cp.sh doc/readme.txt dist
+ 	scripts/substituting_cp.sh scripts/install.sh dist
+ 	scripts/substituting_cp.sh scripts/uninstall.sh dist
+-	cp src/preferences/obj.$(BE_HOST_CPU)/Preferences dist
+-	cp src/filter/obj.$(BE_HOST_CPU)/UniversalScroller dist
++	cp src/preferences/objects/Preferences dist
++	cp src/filter/objects/UniversalScroller dist
+ 	cp LICENSE.txt dist
+ 	chmod +x dist/install.sh
+ 	chmod +x dist/uninstall.sh
+diff --git a/src/filter/Makefile b/src/filter/Makefile
+index db7bf86..8a57778 100644
+--- a/src/filter/Makefile
++++ b/src/filter/Makefile
+@@ -107,7 +107,7 @@ else
+ endif
+ 
+ #	specify additional compiler flags for all files
+-COMPILER_FLAGS =
++COMPILER_FLAGS = -fpermissive
+ 
+ #	specify additional linker flags
+ LINKER_FLAGS =
+diff --git a/src/preferences/Makefile b/src/preferences/Makefile
+index bdf427a..b15fa21 100644
+--- a/src/preferences/Makefile
++++ b/src/preferences/Makefile
+@@ -46,7 +46,7 @@ RSRCS=
+ #		naming scheme you need to specify the path to the library
+ #		and it's name
+ #		library: my_lib.a entry: my_lib.a or path/my_lib.a
+-LIBS= be
++LIBS= be stdc++
```

Comment:
```diff
 -LIBS= be
-+LIBS= be stdc++
++LIBS= be $(STDCPPLIBS)
```
This allows to build on x86_gcc2.
(You don't need to define `STDCPPLIBS`.)

BTW, you might wish to upstream this with the rest of the patchset.

Hmm, relase 4.0.0 is out, it would be better to rename the recipe to `universalscroller-4.0.0_20171227.recipe` or `universalscroller-4.0.1~20171227.recipe`.